letsencrypt/main.py

```diff
@@ -32,6 +32,7 @@
 
 from letsencrypt.display import util as display_util, ops as display_ops
 from letsencrypt.plugins import disco as plugins_disco
+from letsencrypt.plugins import selection as ps
```

Comment:
`ps` isn't clear and while it shortens some of the longer lines in this file, I don't think it's worth using naming conventions that provide little to no information about the purpose of the module.